src/fitnesse/slim/instructions/SystemExitSecurityManager.java

```diff
@@ -35,6 +35,10 @@ private static void tryUpdateSecurityManager(SecurityManager securityManager) {
       System.setSecurityManager(securityManager);
     } catch (SecurityException e) {
       System.err.println("Security manager could not be updated");
+    } catch (UnsupportedOperationException e) {
+      System.err.println("Security manager could not be updated. If you are usin a JDK version >=18, you need to set " +
+        "-Djava.security.manager=allow to allow this. Or use -Dprevent.system.exit=false to disable this feature inside " +
```

Comment:
I think the message can be a bit nicer. Maybe something like: "... -Djava.security.manager=allow to allow this. Or use -Dprevent.system.exit=false to disable the FitNesse feature blocking System.extit() calls and prevent this message."

Reply:
I merged your suggestion, thanks!